cg/cli/upload/base.py

```diff
@@ -51,7 +53,7 @@ def upload(context: click.Context, family_id: Optional[str], restart: bool):
     """Upload results from analyses"""
 
     config_object: CGConfig = context.obj
-    upload_api = MipDNAUploadAPI(config=config_object)  # default upload API
+    upload_api: UploadAPI = MipDNAUploadAPI(config=config_object)  # default upload API
```

Comment:
Hmm, why are we initializing the `MipDNAUploadAPI` and not the `UploadAPI`. It seems confusing looking at the comment and type hint

Reply:
The `MipDNAUploadAPI` is the default one. The variable is later changed depending on the `data_analysis` attribute of the case. With that said I think this module is due for a re-write, but I think it's outside the scope for @diitaz93's second PR